components/MainContainer/MainContainer.tsx

```diff
@@ -11,3 +9,5 @@ export const MainContainer = () => {
     </div>
   );
 };
+
+export default MainContainer;
```

Comment:
same query here as well, any specific reason to use default export?

Reply:
Since the file is used to house only the component, it makes sense to export it by default. Again it's just a personal preference.